### PR TITLE
Feature: Testimonial Block

### PR DESCRIPTION
**Pull request by yahilmadakiya**

### CODE REVIEW

File: blocks/testimonial/index.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+				type: 'color',
+				label: 'Text Color',
+				// placement: 'inspector',
+			},
+		}
+	},
+
+	getEditWrapperProps( attributes ) {
+		const { align } = attributes;
+		if ( 'full' === align ) {
```
Comment (manishsongirkar) [inline blocks/testimonial/index.js]: @yahilmadakiya We also need support for `wide` alignment.

File: blocks/testimonial/index.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+			middleware.fields.bgColor,
+			middleware.fields.textColor,
+			focus && (
+				<BlockControls key="controls">
+					<BlockAlignmentToolbar
+						value={ align }
+						onChange={ ( nextAlign ) => {
+							props.setAttributes( { align: nextAlign } );
+						} }
+						controls={ [ 'full' ] }
```
Comment (manishsongirkar) [inline blocks/testimonial/index.js]: @yahilmadakiya Also add `wide` in controls.

File: blocks/testimonial/index.js (hunk 1 above)
Comment (yahilmadakiya) [reply blocks/testimonial/index.js]: @manishsongirkar  I have added `wide`.

File: blocks/testimonial/index.js (hunk 2 above)
Comment (yahilmadakiya) [reply blocks/testimonial/index.js]: @manishsongirkar  I have added `wide` alignment.

File: blocks/testimonial/index.js [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+		}
+	},
+
+	edit( props, middleware ) {
+		const {
+			attributes: {
+				bgColor,
+				textColor,
+				align,
+				image,
```
Comment (sayedtaqui) [inline blocks/testimonial/index.js]: `image` is redundant here.

File: blocks/testimonial/index.js [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+			},
+			focus,
+		} = props;
+
+		const className = props.className ? props.className : '';
```
Comment (sayedtaqui) [inline blocks/testimonial/index.js]: This is not necessary.

File: blocks/testimonial/index.js [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+const { __ } = wp.i18n;
+
+const {
+	registerBlockType,
+	BlockControls,
+	BlockAlignmentToolbar
+} = wp.blocks;
+
+const {
+	Toolbar
```
Comment (sayedtaqui) [inline blocks/testimonial/index.js]: There is no use of `Toolbar`

File: blocks/testimonial/index.js [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+		} = props;
+
+		const className = props.className ? props.className : '';
+		const hasBackground = bgColor ? ' has-background' : '';
+		const dataAlign = align ? ' align' + align : '';
+
+		return [
+			middleware.inspectorControls,
+			middleware.fields.bgColor,
+			middleware.fields.textColor,
```
Comment (sayedtaqui) [inline blocks/testimonial/index.js]: You don't have to return `middleware.fields.bgColor` and `middleware.fields.textColor`. `middleware.inspectorControls` contains all inspector controls

**Pull request merged**